tests/unit/test_parsing.py

```diff
@@ -93,6 +93,68 @@
             ),
         ),
         ("https://fail@more@at@domain.com", None),
+        (
+            "git@gitlab.com:packit-service/src/libvirt.git",
+            RepoUrl(
+                repo="libvirt",
+                namespace="packit-service/src",
+                hostname="gitlab.com",
+                scheme="git+ssh",
+            ),
```

Comment:
I don't understand, how come this works here and fails for me locally? :D do I have old ogr?

Reply:
it's not fixed yet :D there are 2 questions regarding it
1. do we want to keep user as in actual user? (right now we basically take the namespace, which is also a reason it failed for the url you put in the original issue)
2. shouldn't we default scheme to https (right now it's http)?

Reply:
oh right, the tests are failing :facepalm: 

agreed, definitely default to https

my expectation would be, not sure if it makes sense:
```
assert namespace == "packit-service/src"
assert user == "packit-service"
```

Reply:
ok, I'll try to fix it and probably add comment that we mean user from the git-forge perspective (not the auth, since it is quite confusing that way)

Regarding the `http -> https`, are there any concerns for dependencies, e.g. some git-forge instance that is not running on https?

Reply:
I'd say all run on https and probably even redirect to https